bbot/presets/web/dotnet-audit.yml

```diff
@@ -0,0 +1,21 @@
+description: Comprehensive scan for all IIS/.NET specific modules and module settings
+
+
+include:
+  - iis-shortnames
+
+modules:
+  - httpx
+  - ffuf_shortnames
+  - ffuf
+  - telerik
+  - ajaxpro
+  - dotnetnuke
+
+config:
+  modules:
+    ffuf:
+      extensions: asp,aspx,ashx,asmx
```

Comment:
Is `ascx` common enough to include? I've seen it a couple times

Reply:
It's pretty rare, but I am not opposed to including

Reply:
Any idea why this caused the trufflehog test to fail? :thinking:

Reply:
It's because of this

https://github.com/blacklanternsecurity/bbot/issues/1458

It's fixed in dev but we need to merge it into 2.0